### DISCUSSION TITLE
How to User External DateTime Library like Moment.js or Day.js in Quasar

### DISCUSSION THREAD

**Discussion by mecjos** [quasarframework/quasar#14429] [General - Components / Directives / etc]
Hi. I want to use dayjs as an external date time library.. How can I embed it into quasar app properly?

Thank you.

**Comment by MilosPaunovic** [comment]
You could import it from the boot file.
But, my suggestion would be to first check this out https://quasar.dev/quasar-utils/date-utils

**Reply by mecjos** [reply]
I checked it.. I need export date according to time zone and localization.. quasar date-utils is not enough I think.. I created a boot file I don't know how set up localization and user time zone.
